var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -273,6 +302,9 @@ def setup_build_environment(self, env):
             blas = 'atlas'
         elif spec['blas'].name == 'veclibfort':
             blas = 'accelerate'
+        elif spec['blas'].name == 'fujitsu-ssl2':
+            blas = 'blas'
+            env.append_flags('LDFLAGS', self.spec["blas"].libs.ld_flags)
```

Comment:
Maybe we need to add a `[fujitsu]` section that knows which libs to search for?

Reply:
Thank you for your advice.
I read site.cfg.example.
So, I think that I will set "extra_link_args" on the [blas] and [lapack] sections in case of fujitsu-ssl2 instead of set "LD_FLAGS".
How about this idea ?

Reply:
If that works then it sounds good to me

Reply:
Thank you for your comment.
I fix to use "extra_link_args".

And I divide netlib-lapack and fujitsu-ssl2 on write_library_dirs method.
Because I add "extra_link_args" to only fujitsu-ssl2.

And I fixed "include_dirs" of fujitsu-ssl2 to same as netlib-lapack according to your advice.
I will add  headers method to fujitsu-ssl2 and create other pull request to push it.